Lib/test/pythoninfo.py

```diff
@@ -395,6 +395,24 @@ def collect_sqlite(info_add):
     copy_attributes(info_add, sqlite3, 'sqlite3.%s', attributes)
 
 
+def collect_test_socket(info_add):
+    from test import test_socket
+
+    attributes = (
+        'HAVE_SOCKET_CAN', 'HAVE_SOCKET_CAN_ISOTP', 'HAVE_SOCKET_RDS',
+        'HAVE_SOCKET_ALG', 'HAVE_SOCKET_VSOCK',
+    )
```

Comment:
For be future proof, maybe use dir(socket) and dump all HAVE_* constants?

Reply:
good idea!